liferay/liferay-portal
# LPS-170054 Add error page when an invalid operation occurs

Motivation
=======
For the Segmentation Initiative, we want to introduce a new error page for when an invalid operation occurs for created segments.
[Link to the task](https://issues.liferay.com/browse/LPS-170054)
[Link to the story](https://issues.liferay.com/browse/LPS-168143)

Proposed Solution
========
The proposed solution consists in use an Empty Result message by using its taglib and also adding a ClayButton so it redirects to the SegmentsApp when the error message is shown. Right know we're using a mocked variable 'noSegments' to know when an invalid operation occurs until we have the backend needed for that and the message will appear even when we add a new segment.

Steps to Verify:
----------------
1. Go to People -> Segments.
2. Click on '+' to add a new Segment.
3. Check that the message error appear.

Screencaps
---------------
![Screenshot 2022-12-01 at 10 25 39](https://user-images.githubusercontent.com/91207948/205015830-d0d8edbf-87ac-4f7d-8ffd-7a4a7c961d80.png)
